public/app/plugins/datasource/tempo/resultTransformer.ts

```diff
@@ -161,51 +161,57 @@ function resourceToProcess(resource: collectorTypes.opentelemetryProto.resource.
   return { serviceName, serviceTags };
 }
 
-function getSpanTags(
+function getSpanTags(span: collectorTypes.opentelemetryProto.trace.v1.Span): TraceKeyValuePair[] {
+  const spanTags: TraceKeyValuePair[] = [];
+
+  if (span.attributes) {
+    for (const attribute of span.attributes) {
+      spanTags.push({ key: attribute.key, value: getAttributeValue(attribute.value) });
+    }
+  }
+
+  return spanTags;
+}
+
+function getIntrinsics(
   span: collectorTypes.opentelemetryProto.trace.v1.Span,
   instrumentationLibrary?: collectorTypes.opentelemetryProto.common.v1.InstrumentationLibrary
 ): TraceKeyValuePair[] {
-  const spanTags: TraceKeyValuePair[] = [];
+  const intrinsics: TraceKeyValuePair[] = [];
 
   if (instrumentationLibrary) {
     if (instrumentationLibrary.name) {
-      spanTags.push({ key: 'otel.library.name', value: instrumentationLibrary.name });
+      intrinsics.push({ key: 'otel.library.name', value: instrumentationLibrary.name });
     }
     if (instrumentationLibrary.version) {
-      spanTags.push({ key: 'otel.library.version', value: instrumentationLibrary.version });
-    }
-  }
-
-  if (span.attributes) {
-    for (const attribute of span.attributes) {
-      spanTags.push({ key: attribute.key, value: getAttributeValue(attribute.value) });
+      intrinsics.push({ key: 'otel.library.version', value: instrumentationLibrary.version });
     }
   }
 
   if (span.status) {
-    if (span.status.code && (span.status.code as any) !== SpanStatusCode.UNSET) {
-      spanTags.push({
+    if (span.status.code !== undefined) {
+      intrinsics.push({
         key: 'otel.status_code',
         value: SpanStatusCode[span.status.code],
       });
       if (span.status.message) {
-        spanTags.push({ key: 'otel.status_description', value: span.status.message });
+        intrinsics.push({ key: 'otel.status_description', value: span.status.message });
```

Comment:
So we call these `status.code` & `status.message` in the backend but when transforming here we rename to `otel.status_code` and `otel.status_description` - not fully happy with that approach